src/lib/Renderer.js

```diff
@@ -951,6 +950,8 @@ var Renderer = Base.extend('Renderer', {
             isSelected = isRowSelected || grid.isCellSelectedInRow(r);
         } else if (isGridRow) {
             config = Object.create(cellEvent.properties);
+            alternates = cellEvent.properties.gridRowProperties;
```

Comment:
At first I called the property `alternateRowProperties` because it was just for alternating row property bags. I'll rename the local var `rowProperties`.